chronograf.go

```diff
@@ -528,6 +285,13 @@ type DurationRange struct {
 	Lower string `json:"lower"`
 }
 
+// TimeShift represents a shift to apply to an influxql query's time range
+type TimeShift struct {
+	Label    string `json:"label"`    // user facing description
```

Comment:
Comments in this file usually fit the pattern of `[thing with a capital letter] [description]`